apps/x86/halide/unsharp/unsharp/unsharp.h

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Is there a reason why the codegen results is committed in both: `apps/x86/halide/unsharp/unsharp` and `tests/golden/test_apps/test_unsharp.txt`? I imagine you only need the latter.

Reply:
Oh, that's true. Removed it (and also the blur.c and blur.h files I previously committed).